### PR TITLE
Data-Ingest New Features and attempt at linking to BETY (Work in Progress) 

### PR DESCRIPTION
**Pull request by LiamBurke24**
Added new features to app while waiting for BETY docker issues earlier in the week. Users can now create new input records, db file records, and format records by entering all the inputs in the inputs tab and then selecting `Bety Test`. This should allow us to quickly progress to a final version of this app. It is still necessary for users to manually check that the input was inserted in the various tables. 

## Description

New Features: 
- Hide `Download` button until `doi` is entered. 
- New Radio Buttons ot toggle between download methods 
- Pass Filename from selected file from input method to `Name` field
- Pass new format name and mimetype from the formats box to the inputs box
- `Bety Test` Button to test `dbfiles.input.insert`

## Screenshots

screenshot_2018-07-22 pecan data ingest
screenshot_2018-07-22 pecan data ingest 1
screenshot_2018-07-22 pecan data ingest 2

## Review Time Estimate

- [ ] Immediately
- [x] Within one week
- [ ] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tonygardella** [comment]
@LiamBurke24 general question: What does the Test Bety button do?

**Comment by tonygardella** [comment]
Nevermind I see it adds a test input. Maybe add a way to delete the test input as well.

**Comment by mdietze** [comment]
The latter. Do a dedicated query to create the format, as that is very distinct and quite complicated (once you add the formats_variables). Don't recall if there's an existing function for creating that record, but there's definitely one for querying it which you'll want to look at.  This also makes things simpler --- since a new format is optional (one might be using an existing format), by creating the format record first the input insert never has to deal with the case of whether format exists or not (as it always will)

**Comment by LiamBurke24** [comment]
Want me to include the separate format record query in this PR?

File: shiny/Data-Ingest/server_files/ingest_workflow_svr.R [hunk 1]
```diff
@@ -31,40 +28,103 @@ updateSelectizeInput(session, "InputParentName",  choices = input_names, server
...
+  inputsList$Notes <<- input$InputNotes
+  
+  ## Print List
+  output$summInputs <- renderPrint({inputsList})
+})
+
+
+observeEvent(input$testBety, {
+ output$TestPUT <- renderPrint(inputs$List)
+ inputsList$Path <- "/tmp/RtmpolQsqK/local_tempdir/hf103-01-eddy-2000-01.csv"
```
Comment (mdietze) [inline shiny/Data-Ingest/server_files/ingest_workflow_svr.R]: this seems like an odd choice for a default

Comment (LiamBurke24) [reply shiny/Data-Ingest/server_files/ingest_workflow_svr.R]: NOOOOOOOO. I missed deleting this while testing. :( Hang on.

**Pull request merged**